section2/exercises/ex30.rb

```diff
@@ -0,0 +1,56 @@
+#assigns value to people
```

Comment:
minor style note: add whitespace between the `# Beginning of comment`. A little easier to parse the comments that way